src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -136,9 +136,17 @@
 }
 .color-editor section .color-selection-field .saturation-gradient {
     background-image: -webkit-gradient(linear, 0% 50%, 100% 50%, from(#fff), to(rgba(255,255,255,0)));
+    background-image: -moz-linear-gradient(left, #FFFFFF 0%, rgba(255,255,255,0) 100%);
+    background-image: -o-linear-gradient(left, #FFFFFF 0%, rgba(255,255,255,0) 100%);
+    background-image: -ms-linear-gradient(left, #FFFFFF 0%, rgba(255,255,255,0) 100%);
+    background-image: linear-gradient(left, #FFFFFF 0%, rgba(255,255,255,0) 100%);
```

Comment:
We don't really need to add any prefixed version of linear-gradient, so why not just leave the unprefixed version?

http://caniuse.com/#search=linear-gradient